config/docker/ci-docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+  base:
+    build: ./
```

Comment:
So based on this, 

![screen shot 2018-05-14 at 5 07 51 pm](https://user-images.githubusercontent.com/14099737/40026095-8f603ee2-5799-11e8-8d0e-c4e203e979f5.png)


I think we can just specify `build: .` here and in test. I almost feel like we could drop the base service.

Reply:
I was thinking about this more, and we should be able to specify `build: .` in the test service but we'd want to keep what we have for base

Reply:
I actually simplified this greatly.  Dropped the test service entirely.

Reply:
Nice. So I am wondering, should we change the `base` service to be chrome specific? like `test-chrome`?

Reply:
We could do that.